gestion_base_adresse/install/sql/adresse/10_FUNCTION.sql

```diff
@@ -530,6 +530,34 @@ END;
 $$;
 
 
+-- reset_referencer_com()
+CREATE FUNCTION adresse.reset_referencer_com() RETURNS void
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+
+DROP INDEX IF EXISTS adresse.idx_tmp_centroide_communes;
+CREATE INDEX idx_tmp_centroide_communes ON adresse.commune_deleguee (ST_PointOnSurface(geom));
+TRUNCATE adresse.referencer_com RESTART IDENTITY;
```

Comment:
We should not use a `TRUNCATE` if we do not want to erase the existing data.

We could check if there is a unique constraint on the couple id_com, id_com_deleguee (and/or action if we can have more than one line referencing 2 cities ?). If not, create it and use it for a 

```sql
ON CONFLICT ON CONSTRAINT name_of_the_unique_constraint 
DO NOTHING;
```

Reply:
Or we could add a parameter to the function, `"do_truncate" boolean` to let the user decide if we should truncate first or not ?